### PR TITLE
Feat/delete user account

### PR DESCRIPTION
**Pull request by shreyas1925**
- [x] generate otp and verify otp using mobile number 

- [x] generate otp using aadhaar

- [x]  verify otp using aadhaar number

### CODE REVIEW

**Review by ishit-007 — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -19,6 +19,7 @@
     "joi": "^17.7.0",
     "jose": "^4.11.4",
     "jsonwebtoken": "^9.0.0",
+    "nodemon": "^2.0.21",
```
Comment (souris-dev) [inline package.json:22]: Let's make nodemon a devDependency

File: src/controllers/deleteAccount.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    try {
+        const { otp, txnId, authMethod } = request.body
+        const token = request.headers['x-token'];
+        if(authMethod==='MOBILE_OTP'){
+            const data = await phoneLoginService.verifyMobileOtp(authMethod,txnId,otp,token)
+            response.status(201).json({ message: 'OTP verified', data })
+        }
+        else{
+
+            // not working
```
Comment (souris-dev) [inline src/controllers/deleteAccount.js:37]: // TODO: govt endpoint not working

File: src/services/abdm/delete/aadhar.js [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+        aadhaar:encryptedAadhar,
+        iagree:true
+      },{
+        headers: {
+          'x-token': token
+        }
+      }
+    )
+
+    console.log("data2",data);
```
Comment (souris-dev) [inline src/services/abdm/delete/aadhar.js:23]: Let's remove this

File: src/services/abdm/delete/aadhar.js [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+  } catch (error) {
+    handleAxiosError(error)
+  }
+}
+
+const verifyAadhaarOtp = async (authMethod,txnId, otp,token) => {
+  try {
+    const publicKey = await publicKeyUtil.getPublicKey('v2')
+    const encryptedOtp = await encryptionUtils.encryptData(otp, publicKey)
+    console.log("hello");
```
Comment (souris-dev) [inline src/services/abdm/delete/aadhar.js:35]: Let's remove this console.log

File: src/services/abdm/delete/phone.js [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+      {
+        mobileNumber
+      },{
+        headers: {
+          'x-token': token
+        }
+      }
+      
+    )
+    console.log("data2",data);
```
Comment (souris-dev) [inline src/services/abdm/delete/phone.js:21]: Let's remove this console.log

**Review by souris-dev — CHANGES_REQUESTED** [review]

**Review by SaatwikRishi — APPROVED** [review]

**Review by SaatwikRishi — APPROVED** [review]

**Pull request merged**